### PR TITLE
Task2

### PR DESCRIPTION
**Pull request by andrei-pastukhou**

### CODE REVIEW

File: task2/IE10/.gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+node_modules
```
Comment (vladislavkovaliov) [inline task2/IE10/.gitignore]: Adds, please.

`https://gist.github.com/salcode/10017553`

File: task2/IE10/app/index.html [hunk 2]
```diff
@@ -0,0 +1,44 @@
+<!doctype html>
+<html lang="en">
+<head>
+	<meta http-equiv="x-ua-compatible" content="IE=10">
+	<link rel="stylesheet" href="css/bootstrap.min.css" >
+    <meta name="viewport" content="width=device-width, initial-scale=1" >
+    <style>
+        .card {
+            height: 100%;
+			width: 20rem;
```
Comment (vladislavkovaliov) [inline task2/IE10/app/index.html]: Make good code style.

File: task2/IE10/package.json [hunk 3]
```diff
@@ -0,0 +1,22 @@
+{
+  "name": "task2",
+  "version": "1.0.0",
+  "description": "",
+  "main": "main.js",
+  "scripts": {
```
Comment (vladislavkovaliov) [inline task2/IE10/package.json]: Use, please

```
script: {
  start: "..."
}
```

File: task2/IE10/src/main.js [hunk 4]
```diff
@@ -0,0 +1,205 @@
...
+    // Loads all the notes.
+    for (let key in localStorage) {
+      sources.push(key);
+    }
+
+    const source = sources.join(',');
+
+    if (source) {
+      const url = `${this.config.url}?sources=${source}&sortBy=publishedAt&apiKey=${this.config.apiKey}`;
+      //const req = new Request(url);
```
Comment (vladislavkovaliov) [inline task2/IE10/src/main.js]: Remove comment

Comment (vladislavkovaliov) [inline task2/IE10/src/main.js]: Remove comment

File: task2/IE10/src/main.js [hunk 5]
```diff
@@ -0,0 +1,205 @@
...
+      //if(req) {
```
Comment (vladislavkovaliov) [inline task2/IE10/src/main.js]: Remove comment

File: task2/IE10/src/main.js [hunk 6]
```diff
@@ -0,0 +1,205 @@
...
+        fetch(url)
+          .then((response) => { return response.json();})
+          .then((data) => {
+            data.articles.forEach(this.drawNews.bind(this));
+          });
+      //} else {
```
Comment (vladislavkovaliov) [inline task2/IE10/src/main.js]: Remove comment

File: task2/IE10/src/main.js [hunk 7]
```diff
@@ -0,0 +1,205 @@
...
+   * @param 'news' is object:
+   * {author: 'name of author', description : 'description', publishedAt : 'time when was publish',
+   *     source : {id: , title},
+   *  title: 'title of news', url: 'url to source', urlToImage: 'url to get image to news'}
+  **/
+  drawNews(news) {
+    // Check if not empty.
+    if (Object.keys(news).length !== 0) {
+	  const note = document.createElement('div');
+	  //note.innerHTML = NewsNote.TEMPLATE;
```
Comment (vladislavkovaliov) [inline task2/IE10/src/main.js]: Remove comment

File: task2/IE10/src/main.js [hunk 8]
```diff
@@ -0,0 +1,205 @@
...
+  setNews(note, news) {
+	note.innerHTML = template;
+    note.descriptionElement = note.querySelector('.card-text');
+    note.publishedAtElement = note.querySelector('.publishedAt');
+    note.sourceElement = note.querySelector('.source');
+    note.titleElement = note.querySelector('.card-title');
+    note.urlElement = note.querySelector('.btn');
+    note.imgElement = note.querySelector('.card-img-top');
+	
+	const {name: source_name} = news.source; // name of source
```
Comment (vladislavkovaliov) [inline task2/IE10/src/main.js]: Spaces

File: task2/IE10/src/main.js [hunk 9]
```diff
@@ -0,0 +1,205 @@
...
+    note.descriptionElement.textContent = news.description;
+    note.publishedAtElement.textContent = customFormateDate(formatedDate);
+    note.sourceElement.textContent = source_name;
+    note.titleElement.textContent = news.title;
+  }
+  
+}
+
+
+// Add format function to Data type.
```
Comment (vladislavkovaliov) [inline task2/IE10/src/main.js]: We had discussion about comment style.
Please, make comment the only style on whole the code.

File: task2/plugin/.babelrc [hunk 10]
```diff
@@ -0,0 +1,4 @@
+{
+  "presets": [],
```
Comment (vladislavkovaliov) [inline task2/plugin/.babelrc]: Is it necessary?

Comment (vladislavkovaliov) [inline task2/plugin/.babelrc]: Is it necessary?

File: task2/plugin/.gitignore [hunk 11]
```diff
@@ -0,0 +1 @@
+node_modules
```
Comment (vladislavkovaliov) [inline task2/plugin/.gitignore]: Read comment above, please

File: task2/plugin/package.json [hunk 12]
```diff
@@ -0,0 +1,27 @@
...
+    "babel-runtime": "^6.23.0",
+    "better-log": "^1.3.1"
+  },
+  "devDependencies": {
+    "babel-cli": "^6.26.0",
+    "babel-core": "^6.3.21",
+    "babel-plugin-transform-runtime": "^6.23.0"
+  },
+  "scripts": {
+    "release": "babel src --out-dir dist"
```
Comment (vladislavkovaliov) [inline task2/plugin/package.json]: See comment above

```
script: {
  start: "..."
}
```

Comment (vladislavkovaliov) [inline task2/plugin/package.json]: See comment above

```
script: {
  start: "..."
}
```

**Comment by vladislavkovaliov** [comment]
1. Make readme file that provide the all information about how to run this.(It should be something like instruction)
2. Convert all tabs to spaces (Recommend space number is 2)
3. Fix all comment

Links:
`https://docs.npmjs.com/misc/scripts`
`https://duk.gitbooks.io/airbnb-javascript-guidelines/content/`
`https://stackoverflow.com/questions/22782677/coding-style-in-javascript-why-prefer-2-spaces-not-4-spaces`

Write `done` after comment will be fixed and ping me by skype, please.

**Comment by vladislavkovaliov** [comment]
The style is not purpose of this task but make a sure that style works correctly in IE10.
screen shot 2017-12-01 at 11 44 18 am

File: task2/IE10/.gitignore (hunk 1 above)
Comment (andrei-pastukhou) [reply task2/IE10/.gitignore]: done

File: task2/IE10/app/index.html (hunk 2 above)
Comment (andrei-pastukhou) [reply task2/IE10/app/index.html]: done

File: task2/IE10/package.json (hunk 3 above)
Comment (andrei-pastukhou) [reply task2/IE10/package.json]: done

File: task2/IE10/src/main.js (hunk 4 above)
Comment (andrei-pastukhou) [reply task2/IE10/src/main.js]: done

File: task2/IE10/src/main.js (hunk 5 above)
Comment (andrei-pastukhou) [reply task2/IE10/src/main.js]: done

File: task2/IE10/src/main.js (hunk 6 above)
Comment (andrei-pastukhou) [reply task2/IE10/src/main.js]: done

File: task2/IE10/src/main.js (hunk 7 above)
Comment (andrei-pastukhou) [reply task2/IE10/src/main.js]: done

File: task2/IE10/src/main.js (hunk 8 above)
Comment (andrei-pastukhou) [reply task2/IE10/src/main.js]: done

File: task2/IE10/src/main.js (hunk 9 above)
Comment (andrei-pastukhou) [reply task2/IE10/src/main.js]: Probably done. i'm already understand how write comments inside a class for example:
/**
   * Creates an instance of App.
   *
   * @constructor
   * @this {App}
   * @param {Object} config parameters.
   */

But what i should do with comments outside of class and with comment not to function? For example:
// Create application.
const app = new App(config);

How you can see in that case i didn't change.

File: task2/plugin/.babelrc (hunk 10 above)
Comment (andrei-pastukhou) [reply task2/plugin/.babelrc]: I'm agree with your. At this case it unnecessary )). done

Comment (andrei-pastukhou) [reply task2/plugin/.babelrc]: done

File: task2/plugin/.gitignore (hunk 11 above)
Comment (andrei-pastukhou) [reply task2/plugin/.gitignore]: done

File: task2/plugin/package.json (hunk 12 above)
Comment (andrei-pastukhou) [reply task2/plugin/package.json]: done. But actualy i'm not understand why it so important. I can see only one different in that case we can use only 'npm start' exept 'npm run realese'

Comment (andrei-pastukhou) [reply task2/plugin/package.json]: done

**Pull request merged**